subsys/bluetooth/audio/ascs.c

```diff
@@ -109,12 +109,14 @@ static void ase_status_changed(struct bt_bap_ep *ep, uint8_t old_state, uint8_t
 
 	LOG_DBG("ase %p, ep %p", ase, ep);
 
-	bt_conn_get_info(conn, &conn_info);
+	if (conn != NULL) {
+		bt_conn_get_info(conn, &conn_info);
```

Comment:
Nice change. Could you move the `struct bt_conn_info conn_info;` variable into this scope too? 

```suggestion
	if (conn != NULL) {
		struct bt_conn_info conn_info;
		
		bt_conn_get_info(conn, &conn_info);
```

Reply:
yes, done